src/asr_got_reimagined/main_stdio.py

```
@@ -24,7 +24,7 @@
 
 
 # Now import the server factory
-from src.asr_got_reimagined.server_factory import MCPServerFactory
+from asr_got_reimagined.server_factory import MCPServerFactory
```

Ruff (0.11.9)

Module level import not at top of file

(E402)